deal.II-toolchain/platforms/supported/ubuntu.platform

```diff
@@ -7,6 +7,7 @@
 #   build-essential autoconf automake cmake libtool gfortran python3 \
 #   libboost-all-dev zlib1g-dev \
 #   openmpi-bin openmpi-common libopenmpi-dev \
+#   libgl1-mesa-dev libegl1-mesa-dev \
```

Comment:
Yes, otherwise it complains in `ld` :( Not sure on how else to fix this.
I found the solution here: https://dev.opencascade.org/content/understanding-modulepackage-dependencies-while-building-cmake

Reply:
That is annoying.